packages/suite/src/components/suite/DeviceDisplay/parseTextToPagesAndLines.ts

```diff
@@ -52,10 +53,26 @@ export const parseTextToPagesAndLines = ({
     deviceModel,
     text,
 }: ParseTextToLinesParams): ParseTextToLinesResult => {
+    const getCharsPerRow = (): number => {
+        // T1B1 has varying row length depending on address type.
+        if (deviceModel === DeviceModelInternal.T1B1) {
+            switch (text.length) {
+                // legacy and legacy segwit accounts:
```

Comment:
`legacy and legacy` do I miss something or is it a typo?

Reply:
`legacy` and `legacy segwit`

Reply:
Maybe  legacy accounts and legacy segwit accounts? So its more clear :pray: (in case comment stays)